config/config.go

```diff
@@ -27,7 +25,7 @@ type Config struct {
 	Pool *pool.Config `mapstructure:"pool"`
 	// InternalErrorCode used to override default 500 (InternalServerError) http code
 	InternalErrorCode uint64 `mapstructure:"internal_error_code"`
-	// MaxRequestSize specified max size for payload body in megabytes, set 0 to unlimited.
+	// MaxRequestSize specified max size for payload body in megabytes. 0 = 1GB.
```

Comment:
There is no way to set unlimited body size with 0, as 0 gets converted to 1GB in init defaults.